.github/scripts/trymerge.py

```diff
@@ -296,22 +341,36 @@ def get_committer_author(self, num: int = 0) -> str:
 
     def get_checkrun_conclusions(self) -> Dict[str, str]:
         """ Returns list of checkrun / conclusions """
-        last_commit = self.info["commits"]["nodes"][-1]["commit"]
-        checksuites = last_commit["checkSuites"]
-        # TODO: Implement pagination
-        if bool(checksuites["pageInfo"]["hasNextPage"]):
-            raise RuntimeError("Too many checksuites for commit")
-        rc = {}
-        for node in checksuites["nodes"]:
-            workflow_run = node["workflowRun"]
-            checkruns = node["checkRuns"]
-            if workflow_run is not None:
-                rc[workflow_run["workflow"]["name"]] = node["conclusion"]
-                continue
-            if checkruns is not None:
-                for checkrun_node in checkruns["nodes"]:
-                    rc[checkrun_node["name"]] = checkrun_node["conclusion"]
-        return rc
+        if self.conclusions is not None:
+            return self.conslusiotns
```

Comment:
Typo?

Reply:
Thank you for thorough review, fixed (was running late for lunch)